gym_ignition/envs/ignition_env.py

```diff
@@ -52,6 +52,10 @@ def __init__(self) -> None:
         self.ignenv = factory.make(self.md.getEnvironmentName())
         assert self.ignenv, "Failed to create environment " + self.md.getEnvironmentName()
 
+        # Set the verbosity. Run the script as optimized (-O) to decrease the verbosity.
+        gympp.IgnitionEnvironment.setVerbosity(2)
+        assert(gympp.IgnitionEnvironment.setVerbosity(4) or True)
```

Comment:
Not blocking, but for the future consider that this construct may not be really readable for not Python experts. It took me a while to realize that with `-O` the assert is not being executed.

Reply:
Yes, it's a dirty trick but it works fine :sweat_smile:

Reply:
Now you learnt something new ^^